UnitsNet.Tests/UnitConverterTest.cs

```diff
@@ -77,13 +77,6 @@ public void ConvertByAbbreviation_ConvertsTheValueToGivenUnit(double expectedVal
             Assert.Equal(expectedValue, UnitConverter.ConvertByAbbreviation(inputValue, quantityTypeName, fromUnit, toUnit));
         }
 
-        [Theory]
-        [InlineData(1, "UnknownQuantity", "m", "cm")]
-        public void ConvertByAbbreviation_ThrowsQuantityNotFoundExceptionOnUnknownQuantity(double inputValue, string quantityTypeName, string fromUnit, string toUnit)
```

Comment:
Hm, why was this test case removed? Is it no longer relevant/valid?

Reply:
I see further down why, but I think we can keep the test case and test for UnitNotFoundException instead?

Reply:
It fails because it doesn't throw it any more 😃 Code is unnecessary.

Reply:
But.. I honestly don't know what result to expect when calling this method with these arguments? What results are we getting if not exception?

Reply:
It will also give UnitNotFoundException. I'll add it back and catch that.